mmf/utils/general.py

```diff
@@ -316,3 +316,11 @@ def assert_iterator_finished(iter):
         pass
     else:
         assert False
+
+
+def get_device():
+    return (
```

Comment:
Since this is a function now, can we write it is as a simple if else condition now?